package/doc/sphinx/source/documentation_pages/overview.rst

```diff
@@ -35,7 +35,7 @@ of a protein and the radius of gyration of the backbone atoms are calculated::
         r = cterm.pos - nterm.pos  # end-to-end vector from atom positions
         d = numpy.linalg.norm(r)   # end-to-end distance
         rgyr = bb.radius_of_gyration()  # method of a AtomGroup; updates with each frame
-        print "frame = %d: d = %f Angstroem, Rgyr = %f Angstroem" % (ts.frame, d, rgyr)
+        print("frame = %d: d = %f Angstroem, Rgyr = %f Angstroem" % (ts.frame, d, rgyr))
```

Comment:
while you're here can you turn this print statement into an [f-string](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python)?